modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -358,14 +358,36 @@ function dump_postgresql {
 }
 
 function output_restore_sql {
-  pg_restore -j 2 "${dumpfile}" | sed -r "${sed_cmds}"
+
+  PG_RESTORE_VERSION='13.4'
+  DUMPFILE_VERSION=$(file "${dumpfile}" | awk '{print $NF}')
+
+  case $DUMPFILE_VERSION in
+    v1.13-0)
+      PG_RESTORE_VERSION='9.6.22'
+      ;;
+    v1.14-0)
+      PG_RESTORE_VERSION='13.4'
+      ;;
+    *)
+      >&2 echo "${DUMPFILE_VERSION} is not a supported dump file version"
+      exit 1
+      ;;
+  esac
+  sudo docker run --rm --net=host  \
+	  -v "${tempdir}:/tmp/" \
+	  -v "/root/.pgpass:/tmp/.pgpass" -e PGPASSFILE=/tmp/.pgpass \
```

Comment:
For consistency & neatness, newline for each arg:

```suggestion
	  -v "/root/.pgpass:/tmp/.pgpass" \
	  -e PGPASSFILE=/tmp/.pgpass \
```

Reply:
When sending stuff to a fifo file you need to have a process on the other side that will read from it, that's why the original process is put into the background so we can start the process that will read from the fifo file. Otherwise the initial process would never stop